src/cli/cli.rs

```diff
@@ -485,7 +485,7 @@ pub fn build_cli() -> App<'static, 'static> {
                 .value_name("mode")
                 .help("Specify the terminal color mode: 24bit, 8bit, off, *auto*")
                 .possible_values(&["24bit", "8bit", "off", "auto"])
-                .default_value("auto")
+                .default_value(if output_vt100::try_init().is_ok() {"auto"} else {"off"})
```

Comment:
Instead of modifying the default value, could we add this logic somewhere below, after the command-line parsing logic?

Reply:
It can be done, but I think that's conceptually the wrong place. Making it the default sets the stage for correct behavior without user input and allows the user to choose another behavior.

Choosing 'auto' or forcing color output is a reasonable choice especially if the user wants to pipe the output to a command which can display color via non-VT100 mechanisms.

Doing this after the command-line is processed may override user choice and figuring that out is harder (ie, is the 'auto' we have the default or did the user choose that [again, a viable user choice]).